trappy/systrace.py

```diff
@@ -75,3 +85,20 @@ def trace_hasnt_finished(self):
 
         """
         return lambda x: not x.endswith("</script>\n")
+
+    def format_data(self, unique_word, data_str):
+        if unique_word != 'tracing_mark_write:':
+            return data_str
+
+        match = SYSTRACE_EVENT.match(data_str)
+        if match:
+            data_str = "event={} pid={} func={} data={}".format(
+                match.group('event'), match.group('pid'),
+                match.group('func'), match.group('data'))
+            return data_str
+
+        raise ValueError('Unexpected systrace marker: {}'.format(data_str))
+        return data_str
+
+SYSTRACE_EVENT = re.compile(
```

Comment:
Can you add a comment about what this regular expression does. I know regexes are cool, but really hard to read and get right :smile: